docs/BUILD.md

```diff
@@ -0,0 +1,253 @@
+# Building the App for App Store Submission
+
+First, clone the repo (requires `git` - see [Dependencies](#dependencies)):
+
+```
+git clone https://github.com/WorldHealthOrganization/app.git
+cd app
+```
+
+## iOS
+
+Adapting instructions from [Flutter's iOS deployment docs](https://flutter.dev/docs/deployment/ios).
+
+These instructions assume that an app record has already been added to the App Store Connect account. If you haven't done so already, you can follow [these instructions](https://help.apple.com/app-store-connect/#/dev2cd126805).
+
+### Install Dependencies
+
+`cd` into the `client/flutter/ios` directory and install CocoaPods dependencies (requires `CocoaPods` - see [Dependencies](#dependencies)):
+
+```
+cd client/flutter/ios
+pod install
```

Comment:
@advayDev1 - should we require a specific cocoapods install?

Reply:
See comment at bottom